spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -30,6 +30,38 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile2.1]]
+=== MicroProfile 2.1 === (4Q2018)
+MicroProfile 2.1 is the eigth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+Based on MicroProfile's time-boxed release process, this is an incremental release that includes an update to OpenTracing 1.2.
+
```

Comment:
I would continue to reference the Java EE 8 vs Java EE 7 material (pointing them back at the 1.4 release).

Reply:
Resolved.